charts/external-secrets/templates/draft-content-store-postgresql-branch/postgresql.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: draft-content-store-postgres
```

Comment:
We currently just have the one app account for draft+non-draft so can save some toil/hassle by keeping the same arrangement. (Every additional secretsmanager secret = more manual toil)

Reply:
ok, i'll change that to remove the draft-

Reply:
Done in [2ca4c38](https://github.com/alphagov/govuk-helm-charts/pull/1149/commits/2ca4c389f8fe1b887362d65f546fc6a57957f819)